modules/modelLoader/pixartAlpha/PixArtAlphaLoRALoader.py

```diff
@@ -43,32 +43,32 @@ def load(
             self,
             model: PixArtAlphaModel,
             model_names: ModelNames,
-    ) -> PixArtAlphaModel | None:
+    ):
         stacktraces = []
 
         if model_names.lora == "":
-            return None
+            return
 
         if model_names.lora:
             try:
                 self.__load_internal(model, model_names.lora)
-                return None
+                return
             except Exception:
                 stacktraces.append(traceback.format_exc())
 
             try:
                 self.__load_ckpt(model, model_names.lora)
-                return None
+                return
             except Exception:
                 stacktraces.append(traceback.format_exc())
 
             try:
                 self.__load_safetensors(model, model_names.lora)
-                return None
+                return
             except Exception:
                 stacktraces.append(traceback.format_exc())
         else:
-            return model
+            return
```

Comment:
Is this change intended?

Reply:
The reason I ask is, if we always return `None` in a function, we can use the `return` keyword. If we ever return anything else, we should use `return None` instead, even if the two are equivalent.

Reply:
Yes, it's intentional. This function always returns None. I also changed all the other returns. This was an oversight from an earlier refactor, the return value was the model, now it's not used anymore